web/modules/custom/brms_sor/src/Service/ForestReserveService.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace Drupal\brms_sor\Service;
+
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\node\NodeInterface;
+
+/**
+ * Service for retrieving Forest Reserves data.
+ */
+class ForestReserveService {
```

Comment:
Let's not add a service for this, use a computed/calculated field instead, as this is always going to receive a $node.

Reply:
Removed service and added MasterPolygonKML computed field